kiele

```diff
@@ -81,7 +81,7 @@ run_check() {
 }
 
 run_vm() {
-    iele-vm "$kiele_port" "$kiele_host"
+    exec iele-vm "$kiele_port" "$kiele_host"
```

Comment:
What's this change for?

Reply:
The original test script in the Jenkinsfile had an error, it recorded the pid of kiele so it could kill it at the end of the script. But the kiele script was invoking iele-vm as a new process, so the kill command wasn't killing the vm. The kiele script does nothing after launching the iele-vm, so I made this change which makes the kill command actually work.